pgjdbc/src/main/java/org/postgresql/PGProperty.java

```diff
@@ -600,6 +600,16 @@ public enum PGProperty {
       "0",
       "The timeout value in seconds max(2147484) used for socket read operations."),
 
+  /**
+   * Enable or disable mapping of PG types with TIMEZONE into SQL types with TIMEZONE.
+   * The default is {@code false}
+   */
+  SQL_TYPES_WITH_TIMEZONE(
+      "sqlTypesWithTimezone",
+      "false",
+      "Enable or disable mapping of PG types with TIMEZONE into SQL types with TIMEZONE."
```

Comment:
I might be wrong, however, it looks like the description does not explain the meaning of the property.